pkg/lockfile/match-build-gradle.go

```diff
@@ -48,6 +48,11 @@ func (m BuildGradleMatcher) Match(sourcefile DepFile, packages []PackageDetails)
 			group, artifact, _ := strings.Cut(pkg.Name, ":")
 			// TODO: what to do if, while using extended format, components are split in multiple lines?
 			if strings.Contains(line, group) && strings.Contains(line, artifact) {
+				scope := m.extractingScope(line)
+				if len(scope) > 0 {
+					packages[key].DepGroups = append(packages[key].DepGroups, "runtimeClasspath")
```

Comment:
Why not append `scope` instead of another string literal? The invariant currently is that if scope is not empty, it will be `runtimeClassPath`, so I don't see why we wouldn't use that, instead of introducing two locations that have to be considered if the `extractScope` (pending rename) were to be augmented.

Reply:
you're right, I changed this code a lot before opening the PR, I missed that good catch :)